convoy/settings.py

```diff
@@ -1615,6 +1616,22 @@ def job_environment_variables_secret_id(conf):
     return secid
 
 
+def job_max_task_retry_count(conf):
+    # type: (dict) -> int
+    """Get number of times a task should be retried in a particular job
+    :param dict conf: job configuration object
+    :rtype: int
+    :return: max task retry count
+    """
+    try:
+        max_task_retry_count = int(conf['max_task_retry_count'])
```

Comment:
Drop the implicit conversion. At some point we will attempt to align with json schema and a validator. Also rename to `max_task_retries`